### ISSUE TITLE
2.  trainer.build - metric?

### ISSUE THREAD

**Issue by songhune** [NESAP-STL/nesap-stl#12]
2. get_trainer //   trainers/auto_regressive.py, basic.py, base.py 다니면서 trainer 정보 취득
    trainer/base.py에 evaluate, train_epoch등 미구현 되있다고 말씀하셨는데, 
    basic.py나 auto_regressive에 구현이 되어 있어서 신경 안 써도 될 것 같기도 합니다.

    return AutoRegressiveTrainer    //class임, basic,base 정보 포함
> 이건 close issue

3. trainer.build
    base.py에는 미구현이고 basic.py에 있는데, 
    model = predrnn_pp, optimizer = adam, loss = MSE등의 정보를 가져오는 것 같습니다.
    
    근데 metrics는 뭔지 모르겠네요. predrnn_climate.yaml에 없는 내용이라
    utils/metrics.py가 있기는 한데 잘 모르겠네요.
    애초에 metrics 보고 행렬이라고 생각하고 있었을 정도의 수준이라...
    평가지표? 달성률? 이런 거 같은데

    def get_metrics(metrics_config):
    """Get a dictionary of requested metrics instances"""
        return dict((key, globals()[m]())
            for (key, m) in metrics_config.items())

_Originally posted by @ks1101 in https://github.com/NESAP-STL/nesap-stl/issues/8#issuecomment-885350430_

**Comment by songhune** [comment]
3. metrics:
코드에 Keras의 메트릭 클래스(정확히는 인터페이스)에서 영감을 받았다고 나와있습니다. 
새로운 데이터가 들어가고, 결과가 나오고, state가 reset되는 과정인 거 같은데, 구현이 되어있지 않으므로 일단은 무시해도 될 듯 합니다.
